src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1277,6 +1286,9 @@ def enable_features(cmd, client, resource_group_name, cluster_name, features, ku
                     azrbac_client_id=None, azrbac_client_secret=None, azrbac_skip_authz_check=None, cl_oid=None):
     logger.warning("This operation might take a while...\n")
 
+    # Validate custom token operation
+    custom_token_passed, _ = utils.validate_custom_token(cmd, resource_group_name, "dummyLocation")
```

Comment:
can you clarify why we've added "dummyLocation" here?

Reply:
this is done since if the location is passed as 'None', we would unnecessarily execute the location fetch step in the function 'validate_custom_token'. This step is required only in case of connect and not enable features

Reply:
can we add small comment to explain that?